### PR TITLE
added test application for nativenet transceiver

### PR DESCRIPTION
**Pull request by OlegHahm**
This project can be used with desvirt framework.

### CODE REVIEW

**Comment by LudwigKnuepfer** [comment]
I think defining sender and receiver targets in the Makefile would make sense.. if I got right how this is intended to be used.

**Comment by OlegHahm** [comment]
It was intended to use make with command line parameters, but you're right: dedicated targets in the Makefile would be better. I'll add this.

File: test_nativenet/main.c [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+    mesg.type = SND_PKT;
+    mesg.content.ptr = (char*) &tcmd;
+
+    tcmd.transceivers = TRANSCEIVER_NATIVE;
+    tcmd.data = &p;
+
+    p.length = PACKET_SIZE;
+    p.dst = 0;
+    snd_buffer[0] = SENDER_ADDR;
```
Comment (mehlis) [inline test_nativenet/main.c]: why not `p.src` ?

File: test_nativenet/main.c [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+    puts("Start sending packets");
+    while (1) {
+        snd_buffer[1] = (i & 0xFF00) >> 8;
+        snd_buffer[2] = i & 0x00FF;
```
Comment (mehlis) [inline test_nativenet/main.c]: please define two macros for the right hand code

Comment (mehlis) [inline test_nativenet/main.c]: what does this do? you are using byte 1 and 2 for some magic...the left hand needs clarification

File: test_nativenet/main.c (hunk 1 above)
Comment (OlegHahm) [inline test_nativenet/main.c]: Unfortunately nativenet does not yet havea source address field in the packet header.

File: test_nativenet/main.c (hunk 2 above)
Comment (OlegHahm) [inline test_nativenet/main.c]: Agreed.
(The code is just about sending the 16 bit sequence number as payload.)

File: test_nativenet/main.c (hunk 1 above)
Comment (LudwigKnuepfer) [inline test_nativenet/main.c]: Make an issue and I'll implement it ;-)

Comment (mehlis) [inline test_nativenet/main.c]: did https://github.com/RIOT-OS/projects/issues/29

**Comment by mehlis** [comment]
this test project depends on #29 to be solved.

don't merge now.

**Comment by OlegHahm** [comment]
According to your comments I've added the corresponding Makefile target (receiver is the default target) and changed to use the nativenet source address field in the packet header.

**Comment by OlegHahm** [comment]
Further comments on this?

**Comment by mehlis** [comment]
ok for me

**Comment by LudwigKnuepfer** [comment]
go

**Pull request merged**